components/eamxx/src/control/atmosphere_surface_coupling_exporter.cpp

```diff
@@ -231,9 +231,24 @@ void SurfaceCouplingExporter::initialize_impl (const RunType /* run_type */)
     }
     bool do_export_from_file = are_fields_present and are_files_present;
     if (do_export_from_file) {
+      vos_type export_from_file_reg_names;
+      export_from_file_reg_names = export_from_file_fields;
+      // Check if alternative names have been provided.  This is useful for source data files
+      // that don't use the conventional EAMxx ATM->SRFC variable names.
+      if (export_from_file_params.isParameter("fields_alt_name")) {
+        // The parameter may exist but not be set, so check that it really exists
+        auto tmp = export_from_file_params.get<vos_type>("fields_alt_name");
+        bool are_alt_names_present = (std::find(tmp.begin(),tmp.end(),"NONE") == tmp.end()) and (tmp.size() > 0);
+        if (are_alt_names_present) {
+          EKAT_REQUIRE_MSG(tmp.size()==export_from_file_reg_names.size(),"Error! SurfaceCouplingExport::prescribed_from_file - List of alternative names (fields_alt_name) exists but isn't the same size as the list of fields (fields).");
```

Comment:
Note: in order to get the two names for each entry you can use `ekat::split`:
```
const auto& alt_names = params.get<vos_type>("fields_alt_names");
export_from_file_reg_names = export_from_file_fields;
for (auto entry : alt_names) {
	ekat::strip(entry, ' '); // remove empty spaces, in case user did `a = b` instead of `a=b`.
	auto tokens = ekat::split(entry,'=');
	EKAT_REQUIRE_MSG (tokens.size()==2,"Error! Expected 'X=Y' entry, got '" + entry + "' instead.\n");
	auto it = ekat::find(export_from_file_fields,tokens[1]);
	EKAT_REQUIRE_MSG (it!=export_from_file_fields.end(),
		"Error! RHS name does not match a valid EAMxx field.\n"
		"  - provided name: " + tokens[1] + "\n");
	// Some other check on LHS? Maybe no repetitions, to catch a copy-paste error?

	auto idx = std::distance(export_from_file_fields.begin(),it);
	export_from_file_reg_names[idx] = tokens[0];	
}
```